scripts/import_spreadsheet_dump.sh

```diff
@@ -43,8 +43,8 @@ do
 done
 
 # Fix the auto numbering sequence for installs
-max_install_number=$(($($DOCKER_PG_COMMAND -c "SELECT MAX(install_number) FROM $table_name" -At) + 1))
-$DOCKER_PG_COMMAND -c "ALTER SEQUENCE "$table_name"_install_number_seq RESTART WITH $max_install_number"
+max_install_number=$(($(${DOCKER_PG_COMMAND} -c "SELECT MAX(install_number) FROM meshapi_install" -At) + 1))
+${DOCKER_PG_COMMAND} -c "ALTER SEQUENCE meshapi_install_install_number_seq RESTART WITH ${max_install_number}"
```

Comment:
Hardcoding the variables?

Reply:
What do you mean? The only thing we need to update is the install number sequence, which has a predictable name

Reply:
Gotcha. I was a bit confused because this is used as a variable in the for loop down below. Just wanted to check.

Reply:
Note: there is no longer a for loop down below. This is the only sequence that needs updating, since the PKs are all UUIDs now